Assets/MRTK/Core/Providers/ObjectMeshObserver/SpatialObjectMeshObserver.cs

```diff
@@ -248,6 +248,11 @@ private void ApplyMeshMaterial(SpatialAwarenessMeshObject meshObject)
                 meshObject.Renderer.sharedMaterial = (DisplayOption == SpatialAwarenessMeshDisplayOptions.Visible) ?
                     VisibleMaterial :
                     OcclusionMaterial;
+
+                if (PhysicsMaterial != null)
```

Comment:
Is this going to be a material that developers set as a display option? Or is it on that will always (or never) be present?

Reply:
I really have no idea what you mean here. What is the question or call to action?

Reply:
Do you expect developers to need / desire to switch between the physics material and the visible / occlusion materials? If so, we should ensure that there is proper handling in the DisplayOption property.

Thanks!

Reply:
I don't expect them to do that. It's either on or off. Having said that, the null check should go. You should be able to turn it OFF as well.